composer.json

```diff
@@ -25,7 +25,7 @@
         "doctrine/orm": "^2.11",
         "guzzlehttp/guzzle": "^7.4",
         "monolog/monolog": "^2.3",
-        "ondrejmirtes/better-reflection": "^4.3.81",
+        "ondrejmirtes/better-reflection": "4.3.81",
```

Comment:
Why do we need this change?

Reply:
Because next version `4.4.0` has a constraint `"php": ">=7.4.1,<7.5.0",` that blocks us to use it with php8

Reply:
What if we use 5.3 version https://packagist.org/packages/ondrejmirtes/better-reflection#5.3.0?

Reply:
Updated the major version

Reply:
I've checked b2b demo shop and suite-nonsplit and almost all of them support `~4.3` version. Sorry, I should've check before writing a comment. I think we should go with 4.3.81